src/main/java/org/team1540/robot2024/subsystems/drive/Drivetrain.java

```diff
@@ -60,44 +73,35 @@ public Drivetrain(
                 (targetPose) -> Logger.recordOutput("Odometry/TrajectorySetpoint", targetPose));
     }
 
+    @Override
     public void periodic() {
         gyroIO.updateInputs(gyroInputs);
         Logger.processInputs("Drivetrain/Gyro", gyroInputs);
         for (Module module : modules) {
             module.periodic();
         }
 
-        // Stop moving when disabled
-        if (DriverStation.isDisabled()) {
-            for (Module module : modules) {
-                module.stop();
-            }
-        }
-        // Log empty setpoint states when disabled
         if (DriverStation.isDisabled()) {
```

Comment:
...why?

Reply:
I combined the two branches with the same condition `DriverStation.isDisabled()`

Reply:
Why do you need to call module.stop when disabled was my question

Reply:
Oh idk that's stolen (by you actually)